server/views/partials/overviewPage/alerts/alerts.njk

```diff
@@ -12,7 +12,7 @@
     </div>
     <div class="govuk-summary-list__row--no-border">
         <div class="govuk-!-padding-top-3">
-            <dt class="govuk-!-font-weight-bold govuk-!-margin-bottom-1">Non-associations in {{ user.activeCaseLoad.description }}</dt>
+            <dt class="govuk-!-font-weight-bold govuk-!-margin-bottom-1">Non-associations in {{ prisonerData.prisonName }}</dt>
```

Comment:
Should show where the prisoner is not where the current logged in user active caseload!